internal/util/plugins/makefile.go

```diff
@@ -97,5 +99,10 @@ bundle: kustomize
 # Build the bundle image.
 bundle-build:
 	docker build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
+`
+	// Makefile fragments to add to the base Makefile.
+	makefilePackagemanifests = `
```

Comment:
1. We want users to use bundles, not continue using the older package manifests format. Adding this makefile recipe is confusing because now they see two competing options for packaging their operators for OLM.
1. This addition is outside the scope of this PR, so even if (1) weren't the case, I'd still say no to this addition.